.golangci.yml

```diff
@@ -6,7 +6,9 @@ run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   # Allow 10m, within actions it might take a lot
   timeout: 10m
-
+  # Ignore the beautiful, beautiful Astarte mock file
+  skip-files:
+    - client/astarte_mock.go
```

Comment:
Is this still relevant? Otherwise just remove it

Reply:
It's still relevant because the astarte mock is mostly a big switch that will make beautiful-code linters scream in agony